libs/content/question-bank-ppl/content/020/020.02/020.02.01/020.02.01.01/questions.yaml

```diff
@@ -1,6 +1,108 @@
-- id: HP02002010102
+- id: HPL0003
+  relatedQuestions: []
+  externalIds: []
+  annexes: []
   learningObjectives: 
     - 020.02.01.01.02
-- id: HP02002010103
+  explanation: ""
+  variant:
+    type: simple
+    question:  What is the percentage of oxygen in the atmosphere at 6000 ft? (1,00 P.)
```

Comment:
Future work, we should add "points" as an entry for a question, remove this from the text, and add it there.